megatron/model/transformer.py

```diff
@@ -971,6 +971,9 @@ def parallel_lm_logits(input_, word_embeddings_weight, parallel_output, bias=Non
     else:
         logits_parallel = F.linear(input_parallel, word_embeddings_weight, bias)
 
+    if args is not None and args.use_mup:
+        logits_parallel /= args.mup_m_width
```

Comment:
you may also want to multiply by some tunable scalar value here (like we did in BLTM)

Reply:
I thought that `mup_m_width` was the tunable scalar for the logits here?

Reply:
Is this the part where Y_logits = W_unemb X/m_width?